docs/user/guides/build/email-notifications.rst

```diff
@@ -0,0 +1,40 @@
+How to enable email notifications
+=================================
+
+In this guide, you can learn how to setup build notification via email.
+
+.. note::
+
+   Currently we don't send email notifications on :doc:`builds from pull requests </pull-requests>`.
+
+.. tip::
+    :doc:`/pull-requests`
+        Similarly to email notifications,
+        you can also configure automated feedback for your pull requests.
+
+
```

Comment:
```suggestion
.. tip::
    :doc:`/pull-requests`
        Similarly to email notifications,
        you can also configure automated feedback for your pull requests.


```

We already mentioned PR builds in the above line.

Reply:
True. I'd also rather have a `seealso` for webhooks here - and vice versa on the other howto :+1: